### ISSUE TITLE
Provide additional data for images

### ISSUE THREAD

**Issue by atomicleopard** [alirezamirian/angular-material-lightbox#1]
It would be great to provide a data object for each image, rather than just a url.
For example, I'd like to add a class to each image so that I can overlay info/style them.
Might be useful to pass in a thumbnail while loading etc.
Something like

```
ameLightbox.show([{
  url: "",
  thumbnailUrl: "',
  class: ['my-class']
}])

```

Cool plugin, thanks for sharing it.

**Comment by alirezamirian** [comment]
Yes, custom css class is useful feature and it's quite easy to implement.
Showing thumbnail is also useful and it exist in current todo list.
Thanks for taking time to feedback.
